azure-pipelines.yml

```diff
@@ -8,8 +8,8 @@ jobs:
     vmImage: 'vs2017-win2016'
   strategy:
     matrix:
-      Python36:
-        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
```

Comment:
I think we can embed this directly in the `UsePythonVersion` task; I'm pretty sure this is what's causing the 2 different Azure jobs to run.